components/meter/meter-linear.js

```diff
@@ -0,0 +1,57 @@
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
+
+class MeterLinear extends RtlMixin(LitElement)  {
+	static get styles() {
+		return [ css`
+			.container {
+				position: relative;
```

Comment:
`container` in this case can be replaced by `:host`

Reply:
Aside (given that it can be replaced), don't forget that class names should be fully qualified.

Reply:
whoops